content/blog/2020/07/2020-07-27-machine-learning-plugin-coding-phase2.adoc

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: "Machine Learning Plugin project - Coding Phase 2 blog post"
+tags:
+- machinelearning
+- datascience
+- plugins
+- communitybonding
+- gsoc
+- gsoc2020
+author: loghijiaha
+description: "Coding Phase 2 of Google Summer of Code  2020"
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+image::/images/gsoc/jenkins-gsoc-logo_small.png[align="center"]
+
+Welcome back forks!
+
+This blog post is briefing my coding phase 2 in Jenkins link:/projects/gsoc/2020/projects/machine-learning/[Machine Learning Plugin] for this GSoC 2020.
+
+After successfully passed the evaluation and demo in the phase 1, our team were ahead for facing the challenges in phase 2.
```

Comment:
I think "After successfully passing", or "After having successfully passed"...

And "our team went ahead".